src/encoded/static/components/genome_browser.js

```diff
@@ -563,8 +568,9 @@ class GenomeBrowser extends React.Component {
     }
 
     drawTracks(container) {
-        const highlightLocation = Math.floor((this.state.x1 + this.state.x0) / 2) + 1; // browser uses base 1
-        const highlightString = `${this.state.contig}:${highlightLocation}`;
+        const highlightLocationStart = this.state.highlightLocationStart;
+        const highlightLocationEnd = this.state.highlightLocationEnd;
+        const highlightString = `${this.state.contig}:${highlightLocationStart}-${highlightLocationEnd}`;
```

Comment:
I would probably have skipped lines 571 and 572 and just used this.state in line 573 but this doesn't matter. I think our linting program on Encode would probably suggest destructuring here.